store/store.go

```diff
@@ -332,8 +334,70 @@ func authenticate(r *http.Request, user *auth.UserState) {
 	r.Header.Set("Authorization", buf.String())
 }
 
+// refresh will request a refreshed discharge macaroon for the user
+func refresh(user *auth.UserState) error {
```

Comment:
We already use `refresh` for snaps, maybe we can use `refreshMacaroon` or something to make it slightly more obvious in what context this term is used?